<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrPulseVisualWidget</class>
 <widget class="QWidget" name="SpectrPulseVisualWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QSplitter" name="splVert">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frSpectr">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="DiagSpectr" name="wgtSpectr" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frValues">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QSplitter" name="splHor">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTreeView" name="tvFactors">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QFrame" name="frDiag">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="DynamicDiagram" name="wgtDiag" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicDiagram</class>
   <extends>QWidget</extends>
   <header location="global">dynamicdiagram.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiagSpectr</class>
   <extends>QWidget</extends>
   <header location="global">diagspectr.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>splVert</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>SpectrPulseVisualWidget</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>897</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splHor</sender>
   <signal>splitterMoved(int,int)</signal>
   <receiver>SpectrPulseVisualWidget</receiver>
   <slot>splitterMoved(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>757</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>902</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>splitterMoved(int,int)</slot>
 </slots>
</ui>
